### ISSUE TITLE
Incoming number sent to JS listener

### ISSUE THREAD

**Issue by beradrian** [renanoliveira/cordova-phone-call-trap#2]
It would be useful if instead of the state as a string on the JS listener you'll receive an object with two properties: `state` and `incomingNumber`.

**Comment by leonzinger** [comment]
+1
And caller name if in phone book :)

**Comment by renanoliveira** [comment]
Hi @beradrian and @leonzinger 

Today I have two pull requests about this.
I need study this solution because one used a json object ({"RINGING": "NUMBER}) for all states and other concatenate number on ringing state (RINGING + "NUMBER"). 

Do you have any preference?

**Comment by guillaume-fr** [comment]
It's better to have a JSONObject with the following definition : `{ state:"RINGING", number:"NUMBER}`

It's explicit and avoids breaking changes if you want to add caller name or any other info later.

**Comment by leonzinger** [comment]
I prefer JSON as well.

Thank you!

**Comment by beradrian** [comment]
I agree with @guillaume-fr { state:"RINGING", number:"NUMBER}. JSON is much better.

**Comment by rishii7** [comment]
#7

**Comment by RishabhKothaari** [comment]
Hi @renanoliveira thanks for the plugin , can you pull this fix #7 .
